release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2311,6 +2311,8 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * To assist with troubleshooting, logs collected on boostratp failures by the installer now include IP addresses and routes for the control plane and bootstrap hosts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1956079)[*BZ#1956079*])
 
+* When using a self-signed C2S Amazon Commercial Cloud Services cluster, you cannot pull from or push to an internal image registry. As a workaround, you must set `spec.disableRedirect` to `true` in the `configs.imageregistry/cluster` resource. This lets you pull the image layers from the image registry rather than directly from S3 storage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924568[*BZ#1924568*])
```

Comment:
s/C2S Amazon Commercial Cloud Services cluster/Amazon Commercial Cloud Services cluster(C2S)

Reply:
Yes, of course. Changed, but removed the acronym, seeing as we have no second reference.

Reply:
You are professional, thanks